src/psfmachine/machine.py

```diff
@@ -827,6 +843,9 @@ def build_time_model(self, plot=False, downsample=False):
         downsample: boolean
             If True the `time` and `pos_corr` arrays will be downsampled instead of
             binned.
+        split_models : boolean
```

Comment:
Maybe this could be something like `split_time_series`, `split_model` is harder to understand what is going on. But I don't really feel strongly.

Reply:
oh yeah, `split_model` isn't the best word selection. Maybe something like `time_model_segments` or `split_time_model`.